tests/ts_A_create_table_already_created.erl

```diff
@@ -1,4 +1,3 @@
-%% -*- Mode: Erlang -*-
 %% -------------------------------------------------------------------
```

Comment:
But.. emacs would only need this if you had your sources ending in something other than .erl. And, some odd ones excluding, this line is not used anywhere else in riak.